static/app/actionCreators/onboardingTasks.spec.tsx

```diff
@@ -1,90 +1,55 @@
 import {OrganizationFixture} from 'sentry-fixture/organization';
 
-import {updateOnboardingTask} from 'sentry/actionCreators/onboardingTasks';
-import ConfigStore from 'sentry/stores/configStore';
+import {makeTestQueryClient} from 'sentry-test/queryClient';
+import {renderHook, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {useUpdateOnboardingTasks} from 'sentry/actionCreators/onboardingTasks';
 import OrganizationStore from 'sentry/stores/organizationStore';
 import {OnboardingTaskKey} from 'sentry/types/onboarding';
+import {QueryClientProvider} from 'sentry/utils/queryClient';
+import {OrganizationContext} from 'sentry/views/organizationContext';
 
 describe('actionCreators/onboardingTasks', function () {
-  const api = new MockApiClient();
-  const user = ConfigStore.get('user');
-
   jest.spyOn(OrganizationStore, 'onUpdate');
 
-  describe('updateOnboardingTask', function () {
-    it('Adds the task to the organization when task does not exists', async function () {
-      const detailedOrg = OrganizationFixture();
-
-      // User is not passed into the update request
-      const testTask = {
-        task: OnboardingTaskKey.FIRST_PROJECT,
-        status: 'complete',
-      } as const;
-
-      const mockUpdate = MockApiClient.addMockResponse({
-        url: `/organizations/${detailedOrg.slug}/onboarding-tasks/`,
-        method: 'POST',
-        body: testTask,
-      });
-
-      updateOnboardingTask(api, detailedOrg, testTask);
-      await tick();
-
-      expect(mockUpdate).toHaveBeenCalled();
-
-      expect(OrganizationStore.onUpdate).toHaveBeenCalledWith({
-        onboardingTasks: [{...testTask, user}],
-      });
-    });
```

Comment:
I don't see why we would want to do that, and I believe it is related to the sandbox. However, since the sandbox now uses its own function, we can remove this extra code. Can you please confirm this, @obostjancic ?